rust/docs/NIP17_NIP59_AUDIT.md

```
@@ -0,0 +1,276 @@
+| Go | 14 | ✅ SHA256 | ✅ | ✅ Actual time |
+| Python | 14 | ✅ SHA256 | ✅ | ✅ Actual time |
+| React Native | 14 | ✅ getEventHash | ✅ | ✅ Actual time |
+| Dart | 14 | ✅ SHA256 | ✅ | ✅ Actual time |
+| Electron | 14 | ✅ getEventHash | ✅ | ✅ Actual time |
+
+**✅ FIXED:** All SDKs now use actual timestamp for rumor `created_at`. Only seal and gift-wrap timestamps are randomized per NIP-59.
+
+**Locations:**
+- `android/.../Nip17PayloadBuilder.kt:32-37` - `rumor.copy(createdAt = createdAt)` where `createdAt` is randomized
+- `go/bugstr.go:491-494` - rumor created_at = randomPastTimestamp()
+- `python/bugstr/__init__.py:456-458` - rumor created_at = _random_past_timestamp()
```

markdownlint-cli2 (0.21.0)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)